enigmatry-entry-blueprint-app/src/styles/partials/core/layouts/_visibility.scss

```diff
@@ -8,6 +8,7 @@
 	}
 }
 
+/* stylelint-disable-next-line plugin/stylelint-group-selectors */
```

Comment:
" .hide-on-tablet and .hide-on-mobile have same properties, group them.  plugin/stylelint-group-selectors"

disabled rule for this line, .hide-on-table and .hide-on-mobile don't look the same to me